oe_content.post_update.php

```diff
@@ -129,3 +130,42 @@ function oe_content_post_update_00004(): void {
     }
   }
 }
+
+/**
+ * Add new Author sub entity type with required fields.
+ */
+function oe_content_post_update_00005(): void {
```

Comment:
Let's start using 3.x naming here:
```
 function oe_content_post_update_30005(): void {
 ```

Reply:
As I understand in oe_content component we still using 2. x?

Reply:
Yeah, but this will be merged on 3.x branch.

Reply:
fixed!